### DISCUSSION TITLE
Just show certain parts of the world

### DISCUSSION THREAD

**Discussion by janik06kofler** [banesullivan/localtileserver#176] [Q&A]
Hello, I wanted to make a tileserver from just a certain part of the world, meaning the areas that are on the .tif-file that i used on the server. How can i make the rest of the map invisible or is it not possible?

**Comment by banesullivan** [comment]
I think what you're asking is doable, though would you share an annotated screenshot of what you are hoping to do. Also any code as a starting point too.

**Reply by janik06kofler** [reply]
Hi, thanks for the answer. Basicly, as you can see on the map, i pasted the black/white map into the server. So what i want to do it to hide the whole rest outside the map i pasted. My code so far is the following:

```py
from pathlib import Path

from ipyleaflet import Map
from tileserver import get_leaflet_tile_layer, TileClient, get_leaflet_roi_controls

tile_path = TileClient(Path('C:\\Users\\kofle\\OneDrive\\Documents\\Schule\\FME_11060355_1696228638124_3372\\DownloadService\\FULL25.tif'))


t = get_leaflet_tile_layer(tile_path)


m = Map(center=(46.734096, 11.288801), zoom=200)
m.add_layer(t)
m
```

![Screenshot 2023-10-09 080328](https://github.com/banesullivan/localtileserver/assets/147097867/4ad42a9b-30ec-4b22-bf2d-21db6b8b9e88)

**Reply by banesullivan** [reply]
Sorry for the delay... I think you're asking about hiding the map that's shown around the raster? If so, this would be specific to `ipyleaflet` and you may want to follow up there
